Library/Formula/libmxml.rb

```diff
@@ -20,6 +20,10 @@ def install
                           "--enable-shared",
                           "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    assert shell_output("#{bin}/mxmldoc --version").include?("2.9")
```

Comment:
Instead of making tests version-specific let's just test that they execute with a normal status, which `system` does automatically.

Is there a possibility for a more interesting test? If this is a library, can we build and test a short example?